src/features/forgot-password/ui/ForgotPasswordForm.tsx

```diff
@@ -60,21 +59,13 @@ const ForgotPasswordForm: FC<Props> = props => {
   return (
     <Box {...props}>
       <FormProvider {...form}>
-        <YStack mb={40}>
+        <Box mb={40}>
           <InputField
             accessibilityLabel="forgot-password-email-input"
             name="email"
             placeholder={t('auth:email_address')}
           />
-
-          {error && (
-            <ErrorMessage
-              mode="light"
-              error={{ message: error.evaluatedMessage! }}
-              mt={8}
-            />
-          )}
-        </YStack>
```

Comment:
This is simply removing the error message but it's not looking at the source of the problem.

Did you validate if users are able to reset their password?

Reply:
I think with the fix on BE here https://github.com/ChildMindInstitute/mindlogger-backend-refactor/pull/1146 we can actually keep the error message and close this PR?

Reply:
@jgdev please go ahead and close this PR. This is a [BE issue](https://github.com/ChildMindInstitute/mindlogger-backend-refactor/pull/1146) and has been resolved.